database/sqlite/sqlite_functions.c

```diff
@@ -22,9 +22,8 @@ const char *database_config[] = {
     "multiplier int, divisor int , algorithm int, options text);",
 
     "CREATE TABLE IF NOT EXISTS metadata_migration(filename text, file_size, date_created int);",
-    "CREATE INDEX IF NOT EXISTS ind_d1 on dimension (chart_id, id, name);",
-    "CREATE INDEX IF NOT EXISTS ind_c1 on chart (host_id, id, type, name);",
-    "CREATE INDEX IF NOT EXISTS ind_c2 on chart (host_id, context);",
+    "CREATE INDEX IF NOT EXISTS ind_d2 on dimension (chart_id);",
+    "CREATE INDEX IF NOT EXISTS ind_c3 on chart (host_id);",
```

Comment:
Just want to make sure that skipping ind_d1 is intentional.

Reply:
yes ind_d1 will be dropped